pkg/controller/machine/controller.go

```diff
@@ -347,6 +347,7 @@ func (r *ReconcileMachine) drainNode(machine *machinev1.Machine) error {
 	}
 
 	if nodeIsUnreachable(node) {
+		klog.Infof("Node %q is unreachable, draining will wait for pod deletion for %q seconds and skip after it", node.Name, skipWaitForDeleteTimeoutSeconds)
```

Comment:
may be somehow clarify this is `skipWaitForDeleteTimeoutSeconds` starting when the pod is signalled for deletion, otherwise every time this is logged cause the impression there's `skipWaitForDeleteTimeoutSeconds` left to timeout